runtime/common/src/crowdloan.rs

```diff
@@ -440,8 +444,8 @@ decl_module! {
 
 		/// Withdraw full balance of a contributor to an unsuccessful or off-boarded fund.
 		#[weight = 0]
-		fn withdraw(origin, #[compact] index: FundIndex) {
-			let who = ensure_signed(origin)?;
+		fn withdraw(origin, who: T::AccountId, #[compact] index: FundIndex) {
+			let _ = ensure_signed(origin)?;
```

Comment:
The `?` does already suppress the warning you are trying to avoid AFAIK.

```suggestion
			ensure_signed(origin)?;
```